frost-core/src/lib.rs

```diff
@@ -115,7 +115,7 @@ pub trait Group: Copy + Clone + PartialEq {
     /// A unique byte array buf of fixed length N.
     ///
     /// Little-endian!
-    type Serialization: AsRef<[u8]> + Debug + TryFrom<Vec<u8>>;
+    type Serialization: AsRef<[u8]> + Debug + TryFrom<Vec<u8>> + PartialEq;
```

Comment:
It's better to not add an additional bound to this trait because that will require all users of the crate to implement it even if they don't want to.

In this case we can workaround by manually comparing the elements, see other suggestion

```suggestion
    type Serialization: AsRef<[u8]> + Debug + TryFrom<Vec<u8>>;
```